bitnami/wavefront-adapter-for-istio/templates/istio/adapter.yaml

```diff
@@ -0,0 +1,23 @@
dWxseSBxdWFsaWZpZWQgSXN0aW8gbWV0cmljIGluc3RhbmNlIG5hbWUuCgoPCgcEAAMCAgEEEgRWBFMUCg4KBwQAAwICAQUSA1YECgoOCgcEAAMCAgEBEgNWCxgKDgoHBAADAgIBAxIDVhscCiEKBgQAAwICAhIDWQQSGhIgVGhlIG1ldHJpYyB0eXBlLgoKDwoHBAADAgICBBIEWQRWHQoOCgcEAAMCAgIGEgNZBAgKDgoHBAADAgICARIDWQkNCg4KBwQAAwICAgMSA1kQEQppCgYEAAMCBAASBFwEZwUaWSBEZXNjcmliZXMgbWV0cmljIHR5cGVzIGFzIGluIFtXYXZlZnJvbnRdKGh0dHBzOi8vZG9jcy53YXZlZnJvbnQuY29tL21ldHJpY190eXBlcy5odG1sKS4KCg4KBwQAAwIEAAESA1wJDQo1CggEAAMCBAACABIDXgYSGiQgUmVzZXJ2ZWQgZm9yIHVua25vd24gbWV0cmljIHR5cGVzLgoKEAoJBAADAgQAAgABEgNeBg0KEAoJBAADAgQAAgACEgNeEBEKMgoIBAADAgQAAgESA2AGEBohIFJlcHJlc2VudHMgYSBnYXVnZSBtZXRyaWMgdHlwZS4KChAKCQQAAwIEAAIBARIDYAYLChAKCQQAAwIEAAIBAhIDYA4PCjQKCAQAAwIEAAICEgNiBhIaIyBSZXByZXNlbnRzIGEgY291bnRlciBtZXRyaWMgdHlwZS4KChAKCQQAAwIEAAICARIDYgYNChAKCQQAAwIEAAICAhIDYhARCjoKCAQAAwIEAAIDEgNkBhgaKSBSZXByZXNlbnRzIGEgZGVsdGEgY291bnRlciBtZXRyaWMgdHlwZS4KChAKCQQAAwIEAAIDARIDZAYTChAKCQQAAwIEAAIDAhIDZBYXCjYKCAQAAwIEAAIEEgNmBhQaJSBSZXByZXNlbnRzIGEgaGlzdG9ncmFtIG1ldHJpYyB0eXBlLgoKEAoJBAADAgQAAgQBEgNmBg8KEAoJBAADAgQAAgQCEgNmEhMKVwoGBAADAgIDEgNqBBYaSCBGb3IgbWV0cmljcyB3aXRoIHR5cGUgSElTVE9HUkFNLCB0aGlzIGRlc2NyaWJlcyB0aGUgc2FtcGxlIGRlZmluaXRpb24uCgoPCgcEAAMCAgMEEgRqBGcFCg4KBwQAAwICAwYSA2oECgoOCgcEAAMCAgMBEgNqCxEKDgoHBAADAgIDAxIDahQVCnYKBgQAAwIDABIFbQSFAQUaZSBEZXNjcmliZXMgYSBzYW1wbGUgYXMgaW4gdGhlIFtyY3Jvd2xleS9nby1tZXRyaWNzXShodHRwczovL2dpdGh1Yi5jb20vcmNyb3dsZXkvZ28tbWV0cmljcykgbGlicmFyeS4KCg4KBwQAAwIDAAESA20MEgpCCggEAAMCAwAIABIEbwZ1BxowIE9uZSBvZiB0aGUgc2FtcGxlIGRlZmluaXRpb24gbXVzdCBiZSBzdXBwbGllZC4KChAKCQQAAwIDAAgAARIDbwwWCkIKCAQAAwIDAAIAEgNxCB8aMSBEZWZpbml0aW9uIG9mIGFuIGV4cG9uZW50aWFsbHkgZGVjYXlpbmcgc2FtcGxlLgoKEAoJBAADAgMAAgAGEgNxCBAKEAoJBAADAgMAAgABEgNxERoKEAoJBAADAgMAAgADEgNxHR4KMgoIBAADAgMAAgESA3QIHBohIERlZmluaXRpb24gb2YgYSB1bmlmb3JtIHNhbXBsZS4KChAKCQQAAwIDAAIBBhIDdAgPChAKCQQAAwIDAAIBARIDdBAXChAKCQQAAwIDAAIBAxIDdBobCj8KCAQAAwIDAAMAEgR4Bn4HGi0gRGVzY3JpYmVzIGFuIGV4cG9uZW50aWFsbHkgZGVjYXlpbmcgc2FtcGxlLgoKEAoJBAADAgMAAwABEgN4DhYKKAoKBAADAgMAAwACABIDegghGhUgVGhlIHJlc2Vydm9pciBzaXplLgoKEwoLBAADAgMAAwACAAQSBHoIeBgKEgoLBAADAgMAAwACAAUSA3oIDQoSCgsEAAMCAwADAAIAARIDeg4cChIKCwQAAwIDAAMAAgADEgN6HyAKHwoKBAADAgMAAwACARIDfQgZGgwgVGhlIGFscGhhLgoKEwoLBAADAgMAAwACAQQSBH0IeiEKEgoLBAADAgMAAwACAQUSA30IDgoSCgsEAAMCAwADAAIBARIDfQ8UChIKCwQAAwIDAAMAAgEDEgN9FxgKMQoIBAADAgMAAwESBoEBBoQBBxodIERlc2NyaWJlcyBhIHVuaWZvcm0gc2FtcGxlLgoKEQoJBAADAgMAAwEBEgSBAQ4VCikKCgQAAwIDAAMBAgASBIMBCCEaFSBUaGUgcmVzZXJ2b2lyIHNpemUuCgoVCgsEAAMCAwADAQIABBIGgwEIgQEXChMKCwQAAwIDAAMBAgAFEgSDAQgNChMKCwQAAwIDAAMBAgABEgSDAQ4cChMKCwQAAwIDAAMBAgADEgSDAR8gCjYKBAQAAwMSBokBAowBAxomIERlc2NyaWJlcyB0aGUgbG9nZ2luZyBjb25maWd1cmF0aW9uLgoKDQoFBAADAwESBIkBCg4KVAoGBAADAwIAEgSLAQQVGkQgVGhlIGxvZyBsZXZlbCAob25lIG9mIGVycm9yLCB3YXJuLCBpbmZvLCBkZWJ1Zywgb3Igbm9uZSkuIEV4OiBpbmZvCgoRCgcEAAMDAgAEEgaLAQSJARAKDwoHBAADAwIABRIEiwEECgoPCgcEAAMDAgABEgSLAQsQCg8KBwQAAwMCAAMSBIsBExQKJgoEBAACBhIEjwECEBoYIFRoZSBsb2cgY29uZmlndXJhdGlvbi4KCg8KBQQAAgYEEgaPAQKMAQMKDQoFBAACBgYSBI8BAgYKDQoFBAACBgESBI8BBwsKDQoFBAACBgMSBI8BDg9iBnByb3RvMw==
```

Comment:
I imagine this is the output from the command in line 3?

Reply:
Yes